lib/src/command/add.dart

```diff
@@ -389,13 +391,17 @@ class AddCommand extends PubCommand {
           packagePath, pubspecInformation ?? '^${resultPackage.version}');
     }
 
+    log.fine('Added ^${resultPackage.version} to "$dependencyKey"');
+
     /// Remove the package from dev_dependencies if we are adding it to
     /// dependencies. Refer to [_addPackageToPubspec] for additional discussion.
     if (!isDevelopment &&
         yamlEditor.parseAt(['dev_dependencies', package.name],
                 orElse: () => null) !=
             null) {
       yamlEditor.remove(['dev_dependencies', package.name]);
+
+      log.fine('Removed ^${resultPackage.version} from "dev_dependencies".');
```

Comment:
Again, let's just say package name and `dev_dependencies`, and avoiding mentioning the constraint we removed.. because unless we want to make a lot of logic to log the right thing.